libs/injection/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "bbhnet.injection"
+version = "0.0.1"
+description = "Function for injecting BBH signals into input frames"
+authors = ["Will Benoit" <william.benoit@ligo.org,]
```

Comment:
I think you need to move the close quote to the back and include a close caret > as well

Reply:
Fixed